src/broker/broker.c

```diff
@@ -1965,13 +1967,75 @@ static void module_status_cb (module_t *p, int prev_status, void *arg)
     }
 }
 
+static bool signal_is_deadly (int signum)
+{
+    int deadly_sigs[] = { SIGHUP, SIGINT, SIGQUIT, SIGTERM, SIGALRM };
+    for (int i = 0; i < ARRAY_SIZE (deadly_sigs); i++) {
+        if (signum == deadly_sigs[i])
+            return true;
+    }
+    return false;
+}
```

Comment:
stupid nit, feel free to ignore.  The list of non-deadly signals is shorter.  Could do `signal_is_forwardable()` instead?

Reply:
Well the list of fatal signals is the only one we keep... This could allow us to add other signals that the broker handles without updating the list of "forwardable" signals. IOW, the list of fatal signals is known and smaller than the set of all signals, and may be reduced in the future.